packages/mui-base/src/Progress/Root/ProgressRoot.types.ts

```diff
@@ -0,0 +1,89 @@
+import { BaseUIComponentProps } from '../../utils/types';
+
+export type ProgressContextValue = Omit<UseProgressRootReturnValue, 'getRootProps'> & {
+  ownerState: ProgressRootOwnerState;
+};
+
+export type ProgressRootOwnerState = {
+  direction: ProgressDirection;
+  max: number;
+  min: number;
+};
+
+export interface ProgressRootProps
+  extends UseProgressRootParameters,
+    BaseUIComponentProps<'div', ProgressRootOwnerState> {}
+
+export type ProgressDirection = 'ltr' | 'rtl';
+
+export type ProgressStatus = 'indeterminate' | 'loading' | 'complete';
+
+export interface UseProgressRootParameters {
+  /**
+   * The label for the Indicator component.
+   */
+  'aria-label'?: string;
+  /**
+   * An id or space-separated list of ids of elements that label the Indicator component.
+   */
+  'aria-labelledby'?: string;
+  /**
+   * A string value that provides a human-readable text alternative for the current value of the progress indicator.
+   */
+  'aria-valuetext'?: string;
+  /**
+   * The direction that progress bars fill in
+   * @default 'ltr'
+   */
+  direction?: ProgressDirection;
```

Comment:
We use `dir` elsewhere.

Reply:
The prop is `'direction'` ~ e.g. Tabs: https://master--base-ui.netlify.app/base-ui/react-tabs/components-api/#tabs-root-prop-direction

`'dir'` is the attribute set on the elements via `'direction'` 😬

Reply:
I must have been hallucinating, then, sorry! :)